deps/v8/gypfiles/toolchain.gypi

```diff
@@ -61,7 +61,7 @@
     'v8_android_log_stdout%': 0,
 
     # Force disable libstdc++ debug mode.
-    'disable_glibcxx_debug%': 0,
+    'disable_glibcxx_debug%': 1,
```

Comment:
This change is weird, and I’m not sure what the right place to do it is. It’s required because the `std::vector` that is exposed by `_GLIBCXX_DEBUG` isn’t necessarily ABI-compatible with the one that’s exposed without the define; so either Node would also have to use that flag or V8 needs to disable it by default.

And in general, I’m not sure it’s intentional on @nodejs/v8’s side that debug builds expose a different ABI than release builds…?